PowerShellTools/DebugEngine/VSXHost.cs

```diff
@@ -25,6 +25,7 @@ namespace PowerShellTools.DebugEngine
     using PowerShellTools.Common.ServiceManagement.DebuggingContract;

```

Comment:
Nit: File naming to match the class name? [Fully realizing it's out of scope of this change :)]